stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/list/ChannelItem.kt

```diff
@@ -65,8 +65,8 @@ public fun ChannelItem(
     leadingContent: @Composable RowScope.(ChannelItemState) -> Unit = {
         DefaultChannelAvatar(channelItem = it, currentUser = currentUser)
     },
-    detailsContent: @Composable RowScope.(ChannelItemState) -> Unit = {
-        DefaultChannelDetails(
+    centerContent: @Composable RowScope.(ChannelItemState) -> Unit = {
+        DefaultChannelCenterContent(
```

Comment:
Maybe `DefaultChannelItemX...`

So

`DefaultChannelItemLeadingContent`˛
`DefaultChannelItemCenterContent`˛
`DefaultChannelItemTrailingContent`˛

We have the DefaultChannelAvatar here which should be leading content

Reply:
Yes, agree, there is no issue with having lengthy names. Better for consistency.